### PR TITLE
Rewrite the section on deep learning norms.

### PR DESCRIPTION
**Pull request by srush**
I took a stab at this section and add added in group norm as well using the pooling notation. 

(Forgot how much fun this project is.)

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -343,28 +343,40 @@ \subsubsection{Normalization layers}
...
-by which axes are normalized.
+both by which axes are \textit{standardized} as well as their parameters.
 
-We can define a single generic normalization layer:
+We can define a single generic standardization function as:
 \begin{align*}
-  \nfun{\ax}{XNorm} \colon \mathbb{R}^{\ax} &\rightarrow \mathbb{R}^{\ax} \\
-  \nfun{\ax}{XNorm}(X; \gamma, \beta, \epsilon) &= \frac{X - \nfun{\ax}{mean}(X)}{\sqrt{\nfun{\ax}{var}(X)} + \epsilon} \odot \gamma + \beta
+  \nfun{\ax}{standardize} \colon \mathbb{R}^{\ax} &\rightarrow \mathbb{R}^{\ax} \\
+  \nfun{\ax}{standardize}(X) &= \frac{X - \nfun{\ax}{mean}(X)}{\sqrt{\nfun{\ax}{var}(X) + \epsilon}}
```
Comment (davidweichiang) [inline main.tex:352]: Technically this should depend on \epsilon, right?

File: main.tex [hunk 2]
```diff
@@ -343,28 +343,40 @@ \subsubsection{Normalization layers}
...
 \begin{align*}
-  \gamma, \beta &\in \mathbb{R}^{\ax} \\
   \epsilon &> 0.
 \end{align*}
 
-Now, suppose that the input has three axes:
+Now, consider norm functions each with the same type.
 \begin{align*}
-X &\in \reals^{{\batch \times \chans \times \layer}}
+\text{xNorm} &: \reals^{{\batch \times \chans \times \layer}} \rightarrow \reals^{{\batch \times \chans \times \layer}}
```
Comment (davidweichiang) [inline main.tex]: Here x literally ranges over {Batch,Instance,Layer}, right, so maybe it needs to be typographically distinguished somehow, or maybe there doesn't need to be anything to the left of the colon. (Also, apparently we're supposed to write \colon, not :.)

File: main.tex [hunk 3]
```diff
@@ -343,28 +343,40 @@ \subsubsection{Normalization layers}
...
 \end{align*}
 Then the three kinds of normalization layers can be written as:
 \begin{align*}
-Y &= \nfun{\batch}{XNorm}(X; \gamma, \beta) && \text{batch normalization} \\
-Y &= \nfun{\layer}{XNorm}(X; \gamma, \beta) && \text{instance normalization} \\
-Y &= \nfun{\layer,\chans}{XNorm}(X; \gamma, \beta) && \text{layer normalization}
+\text{BatchNorm}(X; \gamma, \beta) &= \nfun{\batch,\layer}{standardize}(X) \ndot{} \gamma + \beta \text{ where }   \gamma, \beta \in \mathbb{R}^{\chans} \\
```
Comment (davidweichiang) [inline main.tex]: can we insert some &'s to align the side conditions?

File: main.tex (hunk 2 above)
Comment (srush) [reply main.tex]: Dropped the xnorm and colon

File: main.tex (hunk 3 above)
Comment (srush) [reply main.tex]: fixed

File: main.tex (hunk 1 above)
Comment (srush) [reply main.tex:352]: People seem to use a fixed constant. I noted that.

**Pull request merged**